web/src/main/webapp/my-app/layout/partials/home-controls.html

```diff
@@ -0,0 +1,41 @@
+<!-- BETA FILTER OPTION -->
+<div ng-show="$storage.showFilterOption">
+  <md-input-container md-no-float class="md-block home__filter">
+    <md-icon>search</md-icon>
+    <input type="text"
+           class="mp-search"
+           placeholder="Filter items on your home screen"
+           ng-model="filterText"
+           select-on-page-load
+           aria-label="search bar: start typing to filter home page items">
+  </md-input-container>
+</div>
+<!-- SPEED DIAL -->
+<div ng-cloak>
+  <md-fab-speed-dial md-direction="down" class="md-fling md-fab-top-right home__speed-dial">
+    <md-fab-trigger>
+      <md-button aria-label="Home view options" class="md-fab md-accent">
+        <md-icon>settings</md-icon>
```

Comment:
I'm trying to understand a "Settings" speed dial in the context of [Material guidelines about floating action buttons](https://material.io/guidelines/components/buttons-floating-action-button.html#buttons-floating-action-button-floating-action-button).

Looks like FAB is for a primary, positive action to take associated with a page. "Create, Favorite, Share, Navigate, and Explore". And a FAB speed dial is for a bucket of related actions. Is "Options", containing a widget mode toggle and a link to the directory to find more apps, consistent with that? I'm a little concerned that changing view mode on the current page and going elsewhere to find more stuff aren't closely enough related actions to be packaged together in a single FAB speed dial?

This looks a little like the "FAB as overflow menu" discouraged in guidelines? Maybe drop the speed dial and only offer these features in the sidebar menu? Maybe reduce the speed dial to a single FAB focusing on the add-to-home primary action?